src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -2,41 +2,35 @@
 
 import java.lang.reflect.Type;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
-import com.google.appengine.api.datastore.Text;
 import com.google.gson.reflect.TypeToken;
 
-import teammates.common.datatransfer.FeedbackSessionType;
 import teammates.common.datatransfer.attributes.CourseAttributes;
 import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
 import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
 import teammates.common.exception.InvalidParametersException;
 import teammates.common.exception.TeammatesException;
 import teammates.common.util.Assumption;
 import teammates.common.util.Const;
-import teammates.common.util.EmailType;
 import teammates.common.util.JsonUtils;
 import teammates.common.util.Logger;
-import teammates.common.util.SanitizationHelper;
 import teammates.common.util.StatusMessage;
 import teammates.common.util.StatusMessageColor;
 import teammates.common.util.Templates;
 import teammates.common.util.Templates.FeedbackSessionTemplates;
-import teammates.common.util.TimeHelper;
 import teammates.ui.pagedata.InstructorFeedbacksPageData;
 
 public class InstructorFeedbackAddAction extends InstructorFeedbacksPageAction {
 
```

Comment:
This is actually wrong though it's not your fault. Only `Action` and `*AbstractAction` can be extended. So i suggest you to is make a new `*AbstractAction` class, and make the two children classes with common method inheriting that abstract class.

If InstructorFeedbackAddAction uses some methods from InstructorFeedbacksPageAction, then make InstructorFeedbacksPageAction inheriting the new AbstractAction class as well and move the reused methods to that class. 

If it's not clear take a look at my [PR](https://github.com/TEAMMATES/teammates/pull/6480#pullrequestreview-19043342) also the [commits](https://github.com/TEAMMATES/teammates/pull/6830) of this PR.